run.sh

```diff
@@ -1,14 +1,9 @@
-#!/bin/bash
+#!/bin/sh
 
+# Import GPG public keys
 gpg --import /keys/*
 
-cron
-
-# LS_COLORS is set to nothing and for some strange reason crontabs are not allowed to contain such env vars
-unset LS_COLORS
-
-# Create crontab file
-env | cat - > /backup.cron
```

Comment:
For me, the environment variables are available in backup.sh called by cron.

To verify that, I did the following in the switch_to_alpine branch:
- add `echo "S3_BUCKET_NAME: $S3_BUCKET_NAME" >> /test.txt` to backup.sh
- build and run the container with the command from your README.md
- make sure the crontab is installed correctly: `docker exec my-backup crontab -l`
- output: `0 4 * * *  /backup.sh`
- run `docker exec my-backup sh /backup.sh`
- run `docker exec my-backup cat test.txt`
- output: `S3_BUCKET_NAME: myBackupBucket`